test/extensions/filters/listener/common/fuzz/listener_filter_fuzzer.cc

```diff
@@ -10,19 +8,85 @@ void ListenerFilterFuzzer::fuzz(
     Network::ListenerFilter& filter,
     const test::extensions::filters::listener::FilterFuzzTestCase& input) {
   try {
-    fuzzerSetup(input);
+    socket_.setLocalAddress(Network::Utility::resolveUrl(input.sock().local_address()));
+  } catch (const EnvoyException& e) {
+    // If fuzzed local address is malformed or missing, socket's local address will be nullptr
```

Comment:
Do you mean it is ok to be nullptr? If so, maybe you can add a comment saying something like "nullptr is allowed in this fuzzer, so don't need to `return;`."
I think an explanation may help make it more clear
(because a `try-catch` without handling the exception or explaination may look confusing).

Reply:
Yup, the try-catch is empty because I wanted to make the local and remote address fields in the proto optional, so the fuzzer will still run even if those aren't specified (set to nullptr by default). Added some comments in the additional commit to make that more clear.